### PR TITLE
KOGITO-1899: Enable tests in native mode

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA Ticket: https://issues.redhat.com/browse/KOGITO-1899
Description:
Enable the tests in native mode.

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Sgitario** [comment]
I'm running the whole feature locally at the time. Will keep this comment updated.

**Comment by Sgitario** [comment]
Tests ran successfully.

File: test/features/deploy_travel_agency.feature [hunk 1]
```diff
@@ -84,75 +82,73 @@ Feature: Deploy Travel agency service and verify its functionality
...
+    {}
+    """
 
-    Then Service "travels" with process name "travels" is available
+  Then Service "travels" with process name "travels" is available
 
   Examples:
-      | native   | minutes | build-request-cpu | 
-      | disabled | 10      | 1                 | 
+    | native   | minutes | build-request-cpu |
```
Comment (radtriste) [inline test/features/deploy_travel_agency.feature:148]: There are some indentation problems in the lines before (starting line 126), can you have a quick look ?

Comment (Sgitario) [reply test/features/deploy_travel_agency.feature]: Not sure why I see indentations problems locally, but then here is right the opposite (I confirmed I use spaces configuration...). However, I reverted the format changes.

**Pull request merged**